tests/memfile/run-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+for i in {1..64}; do
+	echo "00001111000000001111111100000000" >> tempfile1.dat
+done
+
+mkdir -p temp
+cp tempfile1.dat temp/tempfile2.dat
+
+cd ..
+
+echo "Running from the parent directory with tempfile1.dat"
+../yosys -qp "read_verilog -defer memfile/memory.v; chparam -set MEMFILE \"tempfile1.dat\" memory; synth -top memory"
```

Comment:
Do we need to do `synth` at all, since any error should occur in the front-end, no?

Reply:
I am new with Yosys. I thought that because of I used `-defer`, I needed something to force the use of `chparam`, but seems not. I remove the `synth` and it works.

With yosys compiled without this new feature:

```
Running from the parent directory with content1.dat
memfile/memory.v:15: ERROR: Can not open file `content1.dat` for \$readmemb.
Running from the parent directory with temp/content2.dat
memfile/memory.v:15: ERROR: Can not open file `temp/content2.dat` for \$readmemb.
Running from the parent directory with memfile/temp/content2.dat
Running from the same directory with content1.dat
Running from the same directory with temp/content2.dat
Running from a child directory with content1.dat
../memory.v:15: ERROR: Can not open file `content1.dat` for \$readmemb.
Running from a child directory with temp/content2.dat
../memory.v:15: ERROR: Can not open file `temp/content2.dat` for \$readmemb.
Running from a child directory with content2.dat
../memory.v:15: ERROR: Can not open file `temp/content2.dat` for \$readmemb.
```

With Yosys compiled with this new feature:

```
Running from the parent directory with content1.dat
Running from the parent directory with temp/content2.dat
Running from the parent directory with memfile/temp/content2.dat
Running from the same directory with content1.dat
Running from the same directory with temp/content2.dat
Running from a child directory with content1.dat
Running from a child directory with temp/content2.dat
Running from a child directory with content2.dat
```